ref/addons.md

```diff
@@ -62,6 +63,15 @@ The following figure shows when the different hooks are executed in the life cyc
 ![Hooks execution in the life cycle of a container|471x601](https://assets.ubuntu.com/v1/bc9b1291-addons-reference-hook-order.png)
 
 ### Hook timeouts
-All hooks are subject to a 5 minute timeout to avoid blocking a container for too long.
+All hooks are subject to a 5 minute timeout by default to avoid blocking a container for too long. The timeout can be configured with `hooks.timeout` key in the addon manifest. For example:
+
+```yaml
+name: my-addon
+description: an example addon
+hooks:
+  timeout: 15m
+```
 
 A hook that runs into a timeout exits with an error.
+
+[note type="information" status="Note"]Any value longer than 15 minutes would be considered invalid.[/note]
```

Comment:
```suggestion
[note type="information" status="Note"]Values longer than 15 minutes are considered invalid and ignored.[/note]
```

Reply:
Are they just ignored? Or do they raise an error?

Reply:
The latter one -> this will raise an error.

Reply:
OK great, then we don't need to mention what happens, because users will see it. :)

```suggestion
[note type="information" status="Note"]Values longer than 15 minutes are considered invalid.[/note]
```